yamale/validators/validators.py

```diff
@@ -21,7 +12,236 @@ class String(Validator):
     constraints = [con.LengthMin, con.LengthMax, con.CharacterExclude]
 
     def _is_valid(self, value):
-        return isstr(value)
+        return util.isstr(value)
+
+
+class Pathname(String):
```

Comment:
Don't make any changes yet, I still have to talk to people about Windows support. It's not something we normally do.